front/app/containers/ProjectsShowPage/index.tsx

```diff
@@ -88,32 +81,28 @@ const ProjectsShowPage = ({ project }: Props) => {
     sort: '-start_at',
   });
 
-  const loading = useMemo(() => {
```

Comment:
This data is not used anywhere, so we can remove it.

Reply:
@brentguf I think I remember now - I believe there was a bug with the "Scroll to" behaviour. since it started scrolling to a specific Idea or Event (from id in URL param), but then since things were still loading it shifted midway through and the scroll didn't end up in the right place :thinking: 

So this fix was necessary for making sure everything had loaded first :thinking: